packages/api/app/Validators/UpdateInstitution.js

```diff
@@ -15,6 +17,12 @@ class UpdateInstitution extends BaseValidator {
 			state: 'string',
 			lat: 'string',
 			lng: 'string',
+			email: 'email',
+			phone_number: 'string',
+			website: 'url',
+			type: `string|in:${Object.values(institutionsTypes).join()}`,
+			category: `string|in:${Object.values(institutionsCategories).join()}`,
+			logo_id: 'number|exists:uploads,id',
```

Comment:
Poderemos retirar a logo durante um update?

Reply:
Nesse caso eu coloquei apenas caso o usuário queira atualizar a logo. Se ele passar o logo_id ele checa se ele existe em uploads.

Reply:
No update ele só atualiza, caso o logo_id for null ele não retira